<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightAndChill</class>
 <widget class="QWidget" name="LightAndChill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LightAndChill</string>
  </property>
  <widget class="QComboBox" name="devicesList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enableBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bandsSpin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bands number:</string>
   </property>
  </widget>
  <widget class="QSlider" name="limitSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>22</width>
     <height>130</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
